### DISCUSSION TITLE
Changing styles of scale component and responsive nature of UI

### DISCUSSION THREAD

**Discussion by tejswini-pude** [telekom/scale#1273] [Q&A]
Hi Team,
I want to change style according to my project but I am not able to change any styles.
Example as below:
I want yo make rounded corner to scale text field , I have added style to class of input field  but it does not reflect in UI.
and in scale library I can see breakpoints to make site responsive but classes are not mentioned anywhere.

**Comment by acstll** [comment]
Hey @tejswini-pude, thanks for asking.

Did you take a look at the "Guidelines / Customization and Themes" page in our docs site?

https://telekom.github.io/scale/?path=/docs/guidelines-customization-and-themes--page
